internal/controllers/plans.go

```diff
@@ -0,0 +1,39 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/cyverse/QMS/internal/model"
+	"github.com/labstack/echo"
+)
+
+// swagger:route GET /plans plans listPlans
+// Returns a List all the plans
+// responses:
+//   200: plansResponse
+//   404: RootResponse
+
+func (s Server) GetAllPlans(ctx echo.Context) error {
+	data := []model.Plans{}
+	err := s.GORMDB.Debug().Find(&data).Error
+	if err != nil {
+		return ctx.JSON(http.StatusInternalServerError, model.ErrorResponse(err.Error(), http.StatusInternalServerError))
+	}
+	return ctx.JSON(http.StatusOK, model.SuccessResponse(data, http.StatusOK))
+}
+
+// swagger:route GET /plans/{PlanID} plans listPlansByID
+// Returns a List all the plans
```

Comment:
```suggestion
//
// Get Plan Information
//
// Returns information about a the plan with the given identifier.
```

The first line of the comment is used as the summary. The second line is used as the description. For example, these are some comments from `cyverse-de/requests`:

```go
// swagger:route GET /request-types request-types getRequestTypes
//
// List Request Types
//
// This endpoint lists all of the request types that have been registered, sorted by name.
//
// responses:
//    200: requestTypeListing
//    500: errorResponse
```

The generated Swagger docs are rendered like this:

![image](https://user-images.githubusercontent.com/415143/148468653-b1ef3c0b-f2ae-4fbc-9946-9b2201b0ff07.png)